lib/components/MapMarker.js

```diff
@@ -260,13 +273,20 @@ class MapMarker extends React.Component {
       image = image.uri;
     }
 
+    let icon;
+    if (this.props.icon) {
```

Comment:
I would refactor this to ternary if statement :)

Reply:
+1